test/Negative.test.js

```diff
@@ -1,18 +1,19 @@
+const assert = require('assert');
 const math = require('mathjs');
 
 const flatten = require('../lib/util/flattenOperands');
 const print = require('../lib/util/print');
 
 const Negative = require('../lib/Negative');
 
-const TestUtil = require('./TestUtil');
-
 function testNegate(exprString, outputStr) {
-  const inputStr = Negative.negate(flatten(math.parse(exprString)));
-  TestUtil.testFunctionOutput(print, inputStr, outputStr);
+  it(exprString + ' -> ' + outputStr,  () => {
```

Comment:
thanks so much for fixing all these it things, I've been wanting to get this done since you mentioned it a while ago!